Gemfile

```diff
@@ -4,7 +4,7 @@ ruby File.read('.ruby-version').chomp
 
 # Server
 gem 'puma'
-gem 'rails', '~> 7.0'
+gem 'rails', '~> 7.1.2'
 
 gem 'routing-filter', github: 'svenfuchs/routing-filter'
```

Comment:
Do we want to use our version?

Reply:
Maybe we can go with:
`gem 'rails', '~> 7.1'`